tests/subsys/logging/log_core_additional/src/log_test.c

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Additional test case for log core
+ *
+ */
+
+#include <tc_util.h>
+#include <stdbool.h>
+#include <zephyr.h>
+#include <ztest.h>
+#include <logging/log_backend.h>
+#include <logging/log_ctrl.h>
+#include <logging/log.h>
+
+#define LOG_MODULE_NAME log_test
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, LOG_LEVEL_INF);
+
+struct backend_cb {
```

Comment:
comments?

Reply:
Updated, comment every member of struct backend_cb